src/main/scala/it/pagopa/interop/authorizationmanagement/model/persistence/KeyPersistentBehavior.scala

```diff
@@ -257,6 +257,29 @@ object KeyPersistentBehavior {
           replyTo
         )
 
+      case UpdateAgreementAndEServiceState(
```

Comment:
I do not apply any refactoring here, but it could be a candidate to pattern `case x: X => ...`